src/main/drivers/rangefinder/rangefinder.h

```diff
@@ -17,24 +17,41 @@
 
 #pragma once
 
-#include "drivers/io_types.h"
+#include "common/time.h"
 
-typedef struct sonarConfig_s {
+#include "drivers/io.h"
```

Comment:
Do we really need `io.h` rather than `io_types.h`?

Reply:
Yeah, `io_types.h` is sufficient here.